test/utils/get_dart_version_comment_test.dart

```diff
@@ -1,4 +1,5 @@
 @TestOn('vm')
+@Tags(['dart2'])
```

Comment:
This test explodes on Dart 3 because there are `@dart=2.9` comments that don't work in Dart 3. The lowest version you can use in a comment like that in Dart 3 is 2.12 (null safety).